configs/nuimages/README.md

```diff
@@ -10,6 +10,8 @@ The convert script can be found [here](https://github.com/open-mmlab/mmdetection
 The baseline results include instance segmentation models, e.g., Mask R-CNN, Cascade Mask R-CNN, and HTC.
 We will support panoptic segmentation models in the future.
 
+![demo image](../../resources/nuimages_demo.gif)
```

Comment:
The gif file is too big, may save as JEPG images first and then cat them as gif.

Reply:
The images are JEPG format, but I can compress the gif to reduce the size. I will push it soon.